### DISCUSSION TITLE
The logout icon feels more like login

### DISCUSSION THREAD

**Discussion by oliverpool** [tailwindlabs/heroicons#457] [Ideas]
The current `logout` icons looks like this:
![image](https://user-images.githubusercontent.com/3864879/86263244-3de93f80-bbc1-11ea-8321-6380ba7084c2.png)

Would it be possible to modify the logout icon for the **arrow to point outward** of the door? (and maybe rename the current `logout` icon to `login`)

--- 

To me it looks more like a logout than a login. This impression seems confirmed by a google search:
![image](https://user-images.githubusercontent.com/3864879/86263354-68d39380-bbc1-11ea-9b54-ac9e996b740f.png)

PS: thank you so much for all you great work (refactoringui, tailwindcss...)

**Comment by CharltonSantana** [comment]
Hey, 

I've made some variants using the current logout outline icon as a base. Previewable here: https://codepen.io/charltonsantana/pen/bGEmvEY

```
<svg fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor"><path d="M17 16l4-4m0 0l-4-4 m4 4h-14m5 4v1a3 3 0 01-3 3H6a3 3 0 01-3-3V7a3 3 0 013-3h3a3 3 0 013 3v1"></path></svg>

<svg fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor"><path d="M17 16l4-4m0 0l-4-4 m4 4h-14m5 8 H6a3 3 0 01-3-3V7a3 3 0 013-3h7"></path></svg>

<svg fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor"><path d="M14 16l4-4m0 0l-4-4 m4 4h-11m11 5v1a2 2 0 01-2 2H6a3 3 0 01-3-3V7a3 3 0 013-3h10a2 2 0 0 12 2v1"></path></svg>

<svg fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor"><path d="M17 16l4-4m0 0l-4-4 m4 4h-14m-4 -8 V20"></path></svg>

```

If either @adamwathan / @sschoger approve any of them, I'll go ahead and make a pull request.

**Comment by jsmith** [comment]
Hey @CharltonSantana I don't know if you saw but I think this is being fixed in an update soon? See https://twitter.com/steveschoger/status/1283113613650284545

**Comment by CharltonSantana** [comment]
@jsmith Ahh, my bad :)

**Comment by jsmith** [comment]
Haha I like your designs though :) The second is almost identical to the new design. Just a bit too late it seems like!
